projekt1.Rmd

```diff
@@ -236,8 +239,8 @@ outputData <- data.frame(aggregatedData, minutes)
 ```
 
 
-```{r plot-b2, fig.height=4}
-ggplot(outputData) + geom_point(aes(x = minutes, y = aggregatedData/10000)) + labs(title="Wystąpienia transakcji w czasie", x="Czas", y="Liczba transakcji w jednej minucie[10^4]")
+```{r plot-b2, fig.height=5}
+ggplot(outputData) + geom_point(aes(x = minutes, y = aggregatedData/10000)) + labs(title="Wystąpienia transakcji w czasie", x="Czas, w ciągu wybranych dwóch godzin", y="Liczba transakcji w jednej minucie[10^4]")
```

Comment:
`Czas w ciągu wybranych dwóch godzin`. Jaka ta jednostka? Jaki czas pokazuje? Wciąż nie bardzo rozumiem xd

Reply:
o minuty pewnie chodzi? Warto byłoby to napisać przy wykresie

Reply:
zrobione